contents/handbook/growth/sales/lead-scoring.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Lead scoring
+sidebar: Handbook
+showTitle: true
+---
+
+We calculate lead scores in Salesforce to help us prioritize our inbound book of business.  Put simply, the higher the score the higher value a potential contract with a customer **should** be.
+We use [Clearbit](https://clearbit.com/) to enhance our contact information as it is created and then compute a score out of 70 in [Salesforce](https://posthog.lightning.force.com/lightning/setup/ObjectManager/Lead/FieldsAndRelationships/00NVx000001lSDC/view) based on the following parameters:
+
+- *Employee count* - larger companies are more likely to have a bigger customer base and more usage data to capture.  They are also more likely to need an Enterprise plan.
+- *Ability to pay* - indicates whether a company is likely to pay for a product like PostHog to solve their problems.  This is computed from the estimated company revenue. 
+- *Role* - from experience we sell best to people in an engineering, product or leadership role.
+- *Country* - from experience we know that certain countries have a higher inclination to pay for software so we weight those.
+
+> Note that we also calculate an [ICP score](/handbook/growth/marketing/icp) in Salesforce.  This is more marketing aligned and designed to show us whether we are capturing [who we are building for](/handbook/who-we-are-building-for) as inbound leads.
+
+| Metric         | Value                                                                                                              | Score |
+|----------------|--------------------------------------------------------------------------------------------------------------------|-------|
+| Employee Count | 1-10                                                                                                               | 0     |
+|                | 11-1000                                                                                                            | 10    |
+|                | 1000+                                                                                                              | 20    |
+| Ability to pay | Estimated Revenue $0m-$1m                                                                                          | 0     |
+|                | Estimated Revenue $1m-$10m                                                                                         | 5     |
+|                | Estimated Revenue $10m-$100m                                                                                       | 10    |
+|                | Estimated Revenue $100m+                                                                                           | 20    |
+| Role           | engineering                                                                                                        | 10    |
```

Comment:
Should we upweight people with an 'analytics' type title?

Reply:
This is information from Clearbit which doesn't have that (it's a fixed set of roles and sub-roles)